<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DNMapGeneratorDialog</class>
 <widget class="QDialog" name="DNMapGeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>515</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>603</width>
    <height>515</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Probabilistic DNMap Generation</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>DNMap Probabilistic Generation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Input Settings</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="2">
              <widget class="QDoubleSpinBox" name="_blockedCount1"/>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Double blocked count</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Triple blocked count</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>4 blocked count</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QDoubleSpinBox" name="_blockedCount2"/>
             </item>
             <item row="2" column="2">
              <widget class="QDoubleSpinBox" name="_blockedCount3"/>
             </item>
             <item row="3" column="2">
              <widget class="QDoubleSpinBox" name="_blockedCount4"/>
             </item>
             <item row="4" column="2">
              <widget class="QDoubleSpinBox" name="_blockedCount5"/>
             </item>
             <item row="5" column="2">
              <widget class="QDoubleSpinBox" name="_blockedCount6"/>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Single blocked count</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>6 blocked count</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>5 blocked count</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>7 blocked count</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>8 blocked count</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>9 blocked count</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>10+ blocked count</string>
               </property>
              </widget>
             </item>
             <item row="11" column="1">
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>Number of Tests</string>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QDoubleSpinBox" name="_blockedCount7"/>
             </item>
             <item row="7" column="2">
              <widget class="QDoubleSpinBox" name="_blockedCount8"/>
             </item>
             <item row="8" column="2">
              <widget class="QDoubleSpinBox" name="_blockedCount9"/>
             </item>
             <item row="9" column="2">
              <widget class="QDoubleSpinBox" name="_blockedCount10"/>
             </item>
             <item row="11" column="2">
              <widget class="QDoubleSpinBox" name="numTests"/>
             </item>
             <item row="10" column="1">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Fiber Count</string>
               </property>
              </widget>
             </item>
             <item row="10" column="2">
              <widget class="QDoubleSpinBox" name="_fiberCount"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="toolTip">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The file should contain only one line that contains the counts seperated by space.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Load From File</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Output Settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Number of DNMaps to Generate</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QSpinBox" name="numDNMaps"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Output Location</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="dnMapLocation">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="browseButton">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="toolTip">
              <string>This prefix will be followed by a 3-digit id and .cvs</string>
             </property>
             <property name="text">
              <string>DNMaps Name prefix</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="namePrefix">
             <property name="text">
              <string>DNMap_</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Generate Dead Nozzle Maps</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>DNMapGeneratorDialog</receiver>
   <slot>selectOutputLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DNMapGeneratorDialog</receiver>
   <slot>loadDataFromFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>DNMapGeneratorDialog</receiver>
   <slot>generateMaps()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectOutputLocation()</slot>
  <slot>loadDataFromFile()</slot>
  <slot>generateMaps()</slot>
 </slots>
</ui>
